File: HourRegistrationMobileApp/App/App/App/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace App.Models
{
    public class Project
    {
        public Guid Id { get; set; }
        public String Name { get; set; }
        public String Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: ManagementWebApp/management-web-app/src/app/interfaces/project.ts
import { Team } from './team';
export class Project {
    public constructor(init?: Partial<Project>) {
        Object.assign(this, init);
    }

    Id: string;
    Name: string;
    Description: string;
    StartDate: Date;
    EndDate: Date;
    TeamsOnProject: Array<Team>;
}

export interface Project {
    Id: string;
    Name: string;
    Description: string;
    StartDate: Date;
    EndDate: Date;
    TeamsOnProject: Array<Team>;
}